pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -1,9 +1,15 @@
-{ stdenv, lib, fetchurl, unzip, buildPythonApplication, makeWrapper, wrapGAppsHook
-, qtbase, pyqt5, jinja2, pygments, pyyaml, pypeg2, pyopengl, cssutils, glib_networking
-, asciidoc, docbook_xml_dtd_45, docbook_xsl, libxml2, libxslt
-, gst-plugins-base, gst-plugins-good, gst-plugins-bad, gst-plugins-ugly, gst-libav
+{ stdenv, lib, fetchurl, fetchzip, buildPythonApplication
+, makeWrapper, wrapGAppsHook, qtbase, pyqt5, jinja2, pygments
+, pyyaml, pypeg2, pyopengl, cssutils, glib_networking, asciidoc
+, docbook_xml_dtd_45, docbook_xsl, libxml2, libxslt, attrs
+, gst-plugins-base ? null
+, gst-plugins-good ? null
+, gst-plugins-bad  ? null
+, gst-plugins-ugly ? null
+, gst-libav        ? null
```

Comment:
No need for these to be `? null` I think?

Reply:
I meant to add an assert ` !=null`. I just gave a look at all-packages.nix: the package is called with arguments `inherit (gst_all_1) [...]` (and the same for python packages) so they are always defined? Is it good practice or should I remove it and use `gst_all_1` inside the expression?

Reply:
If it's ok I'll squash the commits.